src/main/java/org/jadice/filetype/matchers/PDFMatcher.java

```diff
@@ -66,6 +67,11 @@ public class PDFMatcher extends Matcher {
 
   public static final String NUMBER_OF_PAGES_KEY = "number-of-pages";
 
+  public static final String CONTAINS_TEXT_KEY = "contains-text";
+  public static final String TEXT_LENGTH_KEY = "text-length";
+  public static final String TEXT_LENGTH_PER_PAGE_KEY = "text-length-per-page";

```

Comment:
Das wären meine 3 Vorschläge an Informationen, die man für Text in einem PDF Dokument analysieren könnte.
Dabei werden die unteren beiden nur gesetzt, falls das PDF wirklich Text enthält.